controllers/openstackclient_controller.go

```diff
@@ -586,8 +588,12 @@ func (r *OpenStackClientReconciler) podCreateOrUpdate(
 		}
 
 		if imageUpdate && isPodUpdate {
-			// init container image is mutable but does nothing so force a delete
-			return &common.ForbiddenPodSpecChangeError{Field: "Spec.InitContainers[0].Image"}
+			// init container image is mutable but does nothing so force a delete by triggering NewForbidden
+			return k8s_errors.NewForbidden(
+				schema.GroupResource{Group: "", Resource: "pods"}, // Specify the group and resource type
+				pod.Name,
+				errors.New("Cannot update Pod spec field - Spec.InitContainers[0].Image"), // Specify the error message
+			)
```

Comment:
with new golangci config, it was complaining on the errors  type cast implementation in the now removed `pkg/common/errors.go`